allofplos/tests/test_unittests.py

```diff
@@ -190,8 +163,8 @@ def test_example_doi2(self):
         self.assertEqual(article.get_fn_dict(), {}, 'get_fn_dict does not transform correctly for {}'.format(article.doi))
         self.assertEqual(article.get_plos_journal(), "PLOS ONE", 'get_plos_journal does not transform correctly for {}'.format(article.doi))
         self.assertEqual(article.get_related_dois(), {'retracted-article': ['10.1371/journal.pone.0035142']}, 'get_related_dois does not transform correctly for {}'.format(article.doi))
-        self.assertEqual(article.get_url()[:100], "http://journals.plos.org/plosone/article/file?id=10.1371/annotation/3155a3e9-5fbe-435c-a07a-e9a4846e", 'get_url does not transform correctly for {}'.format(article.doi))
-        self.assertEqual(article.abstract, "", 'abstract does not transform correctly for {}'.format(article.doi))
+        self.assertEqual(article.get_page(page_type='assetXMLFile'), "http://journals.plos.org/plosone/article/file?id=10.1371/annotation/3155a3e9-5fbe-435c-a07a-e9a4846ec0b6&type=manuscript", 'get_page does not transform correctly for {}'.format(article.doi))
```

Comment:
There is a lot of technical debt building up around these tests(e.g., duplicated code, using assertEqual not assert, everything’s in one file, &c.).  I think before we get too many more feature PRs we should work together on a test refactoring PR. Maybe after this specific PR that becomes a top priority?

Reply:
sure